boards/makerdiary/nrf52832_mdk/nrf52840_m2_defconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Stavros Avramidis <stavros9899@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_NRF52840_M2=y
```

Comment:
this is hwmv1, this board needs adding in hwmv2

Reply:
I also placed it in the folder of another board 🥲😅

Reply:
I amended the commit to be compliant with hmmv2